docs/mmteb/points.md

```diff
@@ -2,6 +2,9 @@
 
 | GitHub            | Total points | New dataset | New task | Dataset annotations | (Bug)fixes | Running Models | Review PR |  Paper Writing | Ideation | Coordination |
 |-------------------| ------------ |-------------| -------- | ------------------- | ---------- | -------------- |-----------| -------------- | -------- | ------------- |
+| KennethEnevoldsen |              | 54          |          |                   8 |         16 |                | 18        |                |          |             5 |
+| x-tabdeveloping   |              | 18          |          |                     |            |                |           |                |          |               |
```

Comment:
Seems like points have been changed (duplicated) for other contributors. please undo these.

Reply:
Oops, thanks for pointing this out!